### DISCUSSION TITLE
Export/import Power Calibration data

### DISCUSSION THREAD

**Discussion by MBadberg** [arendst/Tasmota#15330] [Q&A] [answered]
Is there a way, that i only calibrate one outlet and import that calibration data to my other outlets?

I mean, it is only based on calculations, so there should be a way.

**Comment by sfromis** [comment, marked as answer]
That could not really work, as the calibration data is meant to adjust for potential differences in the components of each outlet, meaning that the data may be different to produce the same readings.

**Reply by MBadberg** [reply]
Yeah. I forgot the fixed values of the resistors inside, that can differ from component to component. PITA :-D
Thanks.

**Comment by barbudor** [comment]
You can print the calibration data in the console (VoltageCal, CurrentCal, ...) and then use them on another identical device.
But that would provide non accurate results
At the end, the calibration process is there to allow accurate values from low accuracy parts

**Reply by MBadberg** [reply]
Yeah. I forgot the fixed values of the resistors inside, that can differ from component to component. PITA :-D
Thanks.
